sdk/ai/Azure.AI.VoiceLive/tests/Azure.AI.VoiceLive.Tests.csproj

```diff
@@ -18,5 +18,6 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
     <PackageReference Include="Moq" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" />
+    <PackageReference Include="Microsoft.CognitiveServices.Speech" VersionOverride="1.46.0" />
```

Comment:
Would you please be so kind as to explain an override is being used here and the plan for removing it for the longer-term?